### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by jxc1690** [Pryaxis/TShock#1921] [General]
Do you need any environment for this? My server is windows2008 open will report an error. thank you

**Comment by jxc1690** [comment]
[http://tiebapic.baidu.com/forum/w%3D580/sign=df6a9fc81d7b02080cc93fe952d8f25f/d0a414385343fbf2037d114ea77eca8064388f06.jpg](url)

**Comment by AxeelAnder** [comment]
升级.net framework版本，至少要4.5.2，建议直接升到最新

**Comment by AxeelAnder** [comment]
另外如果还有问题请来我们qq群聊，先关闭了，谢谢合作

**Comment by jxc1690** [comment]
emmmm 搞得我还去弄机翻译 恶心的翻译  大佬留个QQ群
